components/public-api-server/pkg/apiv1/workspace.go

```diff
@@ -299,6 +299,35 @@ func (s *WorkspaceService) DeleteWorkspace(ctx context.Context, req *connect.Req
 	return connect.NewResponse(&v1.DeleteWorkspaceResponse{}), nil
 }
 
+func (s *WorkspaceService) ListWorkspaceClasses(ctx context.Context, req *connect.Request[v1.ListWorkspaceClassesRequest]) (*connect.Response[v1.ListWorkspaceClassesResponse], error) {
+	conn, err := getConnection(ctx, s.connectionPool)
+	if err != nil {
+		return nil, err
+	}
+
+	classes, err := conn.GetSupportedWorkspaceClasses(ctx)
+	if err != nil {
+		log.Extract(ctx).WithError(err).Error("Failed to get workspace classes.")
+		return nil, proxy.ConvertError(err)
+	}
+
+	res := make([]*v1.WorkspaceClass, 0, len(classes))
+	for _, c := range classes {
+		res = append(res, &v1.WorkspaceClass{
+			Id:          c.ID,
+			Title:       c.DisplayName,
```

Comment:
No need to rename DisplayName -> Title. Let's keep DisplayName for consistency.

Reply:
@svenefftinge I changed it to make it consistent with the `Title` property in the listEditorOptions API - the properties refer to basically the same thing, the primary text string displayed in the UI. If you have a preference for `DisplayName`, I say we rename that API property as well.

Reply:
Yes, ideally we would have consistency about such aspects across our entities. But IdeOptions and WorkspaceClasses are two distinct entities so it's not confusing (just a bit unpretty) to have one using `DisplayName` and the other `Title`. We have other entities where we use `Name`.
But for the very same entity and concept it makes things confusing to rename them on the way through the system. (Btw this also applies to renaming IDEOption to EditorOption, which is the same problem)
TL:DR: There are sometimes good reasons to rename a thing but when we do it we should have a strategy to rename completely. And since that can be painful some renames are not with it.

Reply:
@svenefftinge understood. I think that looking forward, it would be wonderful to have some internal naming conventions which would be universal. I'd say the consistency of the external naming (public api) is more important than the internal one (server api), but it's a difficult problem to get right without thinking about it before implementation.

I'll keep this on the back of my mind and maybe we can make some changes before marking the API as stable.

Reply:
Changed it now. Thanks again for the input